css/custom.css

```diff
@@ -359,3 +359,80 @@ footer {
 .color-outreachy:active {
 	background-color: #58cfe5;
 }
+/* ---------------------------Animated Hamburger Menu--------------------- */
+.animated-icon {
+    width: 30px;
+    height: 20px;
+    position: relative;
+    margin: 0px;
+    -webkit-transform: rotate(0deg);
+    -moz-transform: rotate(0deg);
+    -o-transform: rotate(0deg);
+    transform: rotate(0deg);
+    -webkit-transition: .5s ease-in-out;
+    -moz-transition: .5s ease-in-out;
+    -o-transition: .5s ease-in-out;
+    transition: .5s ease-in-out;
+    cursor: pointer;
+    }
```

Comment:
All the closing parentheses don’t need to be indented. :)

Reply:
I indented, please check it.